backend/src/server/routes/v1/secret-sharing-router.ts

```diff
@@ -63,6 +64,7 @@ export const registerSecretSharingRouter = async (server: FastifyZodProvider) =>
       response: {
         200: SecretSharingSchema.pick({
           encryptedValue: true,
+          password: true,
```

Comment:
I don't think password should be returned here in response. That defeats the whole purpose of password protection

Reply:
true, fixed